### PR TITLE
Stm32mp1 mfgtool/RPMB patches

### PR DESCRIPTION
**Pull request by igoropaniuk**
Please do not submit a Pull Request via github.  Our project makes use of
mailing lists for patch submission and review.  For more details please
see https://www.denx.de/wiki/U-Boot/Patches

The only exception to this is in order to trigger a CI loop on Azure prior
to posting of patches.

### CODE REVIEW

File: drivers/fastboot/Kconfig [hunk 1]
```diff
@@ -218,6 +218,12 @@ config FASTBOOT_CMD_OEM_BOOTBUS
 	  Add support for the "oem bootbus" command from a client. This set
 	  the mmc boot configuration for the selecting eMMC device.
 
+config FASTBOOT_CMD_OEM_UCMD
+	bool "Enable the 'oem ucmd' command"
```
Comment (mike-scott) [inline drivers/fastboot/Kconfig:222]: After I added this as a subscriber patch, I wondered if this shouldn't be default n.

@ricardosalveti ?

**Comment by mike-scott** [comment]
@igoropaniuk Thank you for getting this ready!

**Review by MrCry0 — APPROVED** [review]
LGTM

File: drivers/fastboot/Kconfig (hunk 1 above)
Comment (igoropaniuk) [reply drivers/fastboot/Kconfig:222]: ^^ @ricardosalveti

Comment (MrCry0) [reply drivers/fastboot/Kconfig:222]: It's disabled by default.

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Pull request merged**